src/components/shared/TooltipHeaderWithDuration.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  duration: number,
+  title: string,
+  className?: string,
+};
+
+export default class TooltipHeaderWithDuration extends PureComponent {
+  props: Props;
+
+  render(): React$Element<*> {
+    console.log('PLOP3');
+    let { duration } = this.props;
```

Comment:
yeah, maybe it's better.

I usually don't like `utils` so much as we end up with a pile of unrelated stuff. I guess I've an OO background and that's why I found it was better to encapsulate it in a proper component.

I'll try what you suggest and see how it feels :)